src/sourmash/sbt.py

```diff
@@ -666,33 +702,64 @@ def save(self, path, storage=None, sparseness=0.0, structure_only=False):
                 else:
                     data['filename'] = node.save(data['filename'])
 
+
             if isinstance(node, Node):
                 nodes[i] = data
             else:
                 leaves[i] = data
 
+                row = node.make_manifest_row(data['filename'])
+                if row:
+                    manifest_rows.append(row)
+
             if n % 100 == 0:
                 notify("{} of {} nodes saved".format(n+1, total_nodes), end='\r')
 
-        # now, save the index file.
+        # now, save the index file and manifests.
         #
-        # for zipfiles, it gets saved in the zip file.
+        # for zipfiles, they get saved in the zip file.
         # for FSStorage, we use the storage.save function.
         #
         # for everything else (Redis, IPFS), the index gets saved locally.
         # the nodes/leaves are saved/loaded from the datatabase, and
         # the index is used to get their names for loading.
+        # (CTB: manifests are not yet supported for Redis and IPFS)
         #
         notify("Finished saving nodes, now saving SBT index file.")
         info['nodes'] = nodes
         info['signatures'] = leaves
 
+        # finish constructing manifest object & save
+        manifest = CollectionManifest(manifest_rows)
+        manifest_name = f"{name}.manifest.csv"
```

Comment:
Something larger to discuss around `Storage`, but should this go into another dir (`manifests/`?)? SBT originally contained all nodes inside the same dir, but if `internal/` and `signatures/` is used then the sigs can be reused for other indices (linear/LCA) in the same `Storage`. The same is true for manifests, I guess? (and would also apply to the actual Index descriptions, putting SBT and LCA into `indices/`)

Or are manifests unique per Storage, and always contain a complete description of all signatures under `signatures/`?

```suggestion
        manifest_name = f"manifests/{name}.manifest.csv"
```

Reply:
these are questions that I am asking myself, no clear idea!

Reply:
one quick hot take, indirection is good - I like the approach in https://github.com/sourmash-bio/sourmash/pull/1597, where the manifest location is specified in a JSON file.

Reply:
(Did you link back to this PR?)

"in a JSON file", or "in the SBT JSON file"? Because both can be useful, for different contexts. Former would be an equivalent to the collection description (as we have SBT or LCA descriptions), latter still allows distributing IPFS SBTs with a single JSON file.

Reply:
> (Did you link back to this PR?)

oh, whoops, thought this was a comment on #1590, sorry!
 
> "in a JSON file", or "in the SBT JSON file"? Because both can be useful, for different contexts. Former would be an equivalent to the collection description (as we have SBT or LCA descriptions), latter still allows distributing IPFS SBTs with a single JSON file.

what do you mean by "SBT / LCA descriptions"?

In the SBT JSON file is what we're currently doing in this PR. I'm not quite sure how to think about  collections and indexes... For the moment it seems like maybe the simplest path forward is to default to good support for .zip (standardized manifest name) and .sbt.zip (manifest linked in .sbt.json file), while continuing to support .sbt.json files for IPFS and REDIS.

Reply:
(without pushing the edge on anything else until we have at least one or two good examples of e.g. multiple indexes in a single file)

but maybe that's me being lazy...

Reply:
> what do you mean by "SBT / LCA descriptions"?

the `.sbt.json`/`.lca.json` files. "Descriptions" because they describe the structure of the tree (in the SBT case), but the LCA case is the actual DB at the moment, so it is not really a description... But it would be useful if it was, because then we could manage the hash->dataset mapping in other ways, like splitting into multiple files within a ZipStorage (and maybe using the banding from #1594 to do the splitting?)

> In the SBT JSON file is what we're currently doing in this PR. I'm not quite sure how to think about collections and indexes... For the moment it seems like maybe the simplest path forward is to default to good support for .zip (standardized manifest name) and .sbt.zip (manifest linked in .sbt.json file), while continuing to support .sbt.json files for IPFS and REDIS.

We are using Zip files as key-value stores (paths as keys, sigs/internal nodes/SBT JSON/manifests as values), so that is not hard to support with IPFS/Redis.

(and just to be clear, the manifest location in the JSON file works great for key-value stores: in IPFS it would point to another address, in Zip files we can read them from the file, in Redis it would be another key to check).

Reply:
> > what do you mean by "SBT / LCA descriptions"?
> 
> the `.sbt.json`/`.lca.json` files. "Descriptions" because they describe the structure of the tree (in the SBT case), but the LCA case is the actual DB at the moment, so it is not really a description... But it would be useful if it was, because then we could manage the hash->dataset mapping in other ways, like splitting into multiple files within a ZipStorage (and maybe using the banding from #1594 to do the splitting?)

yes! see https://github.com/sourmash-bio/sourmash/issues/1591 for reverse indices that point to signatures.

> > In the SBT JSON file is what we're currently doing in this PR. I'm not quite sure how to think about collections and indexes... For the moment it seems like maybe the simplest path forward is to default to good support for .zip (standardized manifest name) and .sbt.zip (manifest linked in .sbt.json file), while continuing to support .sbt.json files for IPFS and REDIS.
> 
> We are using Zip files as key-value stores (paths as keys, sigs/internal nodes/SBT JSON/manifests as values), so that is not hard to support with IPFS/Redis.

yep.

> (and just to be clear, the manifest location in the JSON file works great for key-value stores: in IPFS it would point to another address, in Zip files we can read them from the file, in Redis it would be another key to check).

yes!